docs/src/api/class-tracing.md

```diff
@@ -3,6 +3,10 @@
 
 API for collecting and saving Playwright traces. Playwright traces can be opened in [Trace Viewer](../trace-viewer.md) after Playwright script runs.
 
+:::note
+The `context.tracing` API records different information than the automatic tracing enabled through [Playwright Test configuration](https://playwright.dev/docs/api/class-testoptions#test-options-trace). This API records browser operations and network activity, but does not capture test assertions (`expect` calls).
```

Comment:
While I like the explanation, I think that adding a suggestion "you should use config option instead" would benefit more users, because they will immediately understand what to do.